internal/command/ssh/console_windows.go

```diff
@@ -1,30 +1,71 @@
-//go:build windows
-// +build windows
```

Comment:
did you meant to remove these?

Reply:
I'll add them back for clarity and consistency. I took them out because they were also implied by the filename, but that is a bit more brittle and magical.